Could take a look at [a test failure on darwin](https://logs.nix.ci/?key=nixos/nixpkgs.71992&attempt_id=acba5f73-bd26-4385-a2a1-5584773d315c)?

```
E               subprocess.CalledProcessError: Command '['git', 'rev-parse', '--abbrev-ref', 'HEAD']' returned non-zero exit status 128.
/nix/store/jn8dic1klsd1d550wb682ssnq9wsffg5-python3-3.7.5/lib/python3.7/subprocess.py:512: CalledProcessError
----------------------------- Captured stdout call -----------------------------
Initialized empty Git repository in /private/tmp/nix-build-mu-repo-1.8.0.drv-0/source/test_temp_dir/lib/.git/
Initialized empty Git repository in /private/tmp/nix-build-mu-repo-1.8.0.drv-0/source/test_temp_dir/app/.git/
----------------------------- Captured stderr call -----------------------------
fatal: ambiguous argument 'HEAD': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'
```

--- a/pkgs/applications/misc/mu-repo/default.nix
+++ b/pkgs/applications/misc/mu-repo/default.nix
@@ -1,0 +1,26 @@
+{ lib, fetchFromGitHub, buildPythonApplication, pytest, git }:
+
+buildPythonApplication rec {
+  pname = "mu-repo";
+  version = "1.8.0";
+
+  src = fetchFromGitHub {
+    owner = "fabioz";
+    repo = pname;
+    rev = with lib;
+      "mu_repo_" + concatStringsSep "_" (splitVersion version);
+    sha256 = "1dxfggzbhiips0ww2s93yba9842ycp0i3x2i8vvcx0vgicv3rv6f";
+  };
+
+  checkInputs = [ pytest git ];
+  # disable test which assumes it's a git repo
+  checkPhase = "py.test mu_repo --ignore=mu_repo/tests/test_checkout.py";
+
+  meta = with lib; {
+    description = "Tool to help in dealing with multiple git repositories";
+    homepage = "http://fabioz.github.io/mu-repo/";
+    license = licenses.gpl3;
+    platforms = platforms.unix;
+    maintainers = with maintainers; [ sikmir ];
+  };
+}

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -19808,6 +19808,8 @@ in
 
   mtpaint = callPackage ../applications/graphics/mtpaint { };
 
+  mu-repo = python3Packages.callPackage ../applications/misc/mu-repo { };
+
   mucommander = callPackage ../applications/misc/mucommander { };
 
   multimarkdown = callPackage ../tools/typesetting/multimarkdown { };

